### ISSUE TITLE
Right side bar triangle button color

### ISSUE THREAD

**Issue by bergart** [nextcloud/spreed#503]
In my opinion, the button is not noticeable. If a user do not know this, it will be difficult to confirm a call.

![selection_037](https://user-images.githubusercontent.com/22152652/33167846-32128498-d03f-11e7-93b1-624a4f8a7f12.png)

probably better in `#0082c9`?

![selection_038](https://user-images.githubusercontent.com/22152652/33167864-3c0a7eec-d03f-11e7-915a-d21643c9d75f.png)

**Comment by nickvergessen** [comment]
Also a nice idea, but also see #461

**Comment by danxuliu** [comment]
>  In my opinion, the button is not noticeable.

I agree (and I am the one who added the button :-P ).

>  it will be difficult to confirm a call.

Mmmm, now that you mention that I think that currently if a user is not in a call she does not know if others have joined it; that should be fixed too :-)

>  probably better in #0082c9?

The reason to not use that color for the triangle was that I intended to reserve it to notify the user if something was going on in the sidebar when it was closed (for example, if new messages were received in the chat). The idea was to "blink" the triangle a couple of times and then keep it in blue until the sidebar was opened; an alternative would be to show it in blue and blink it in white until it was opened, but I think that a constant blink would be too annoying, and keeping it white would simply reverse the current issue.

Maybe an option would be to blink it just once when the sidebar is enabled or closed to draw the attention of the user to the triangle?

Opening the sidebar by default as mentioned by @nickvergessen would help too, but I think that would mitigate instead of solve this issue :-(
